src/encoded/static/components/viz/AboutPageVisualizations/Alluvial/data/data_v2.json

```diff
@@ -0,0 +1,157 @@
+    {
```

Comment:
Might be good to change the name of this file - `data_v2` isn't super descriptive. Is this a second version of the `alluvial_data.json`? If so, could retaining the previous name make sense, since the old file has been deleted?

Reply:
Yes I agree, I'll change to "alluvial_data.json"